### DISCUSSION TITLE
helm chart for nats-operator

### DISCUSSION THREAD

**Discussion by ekawas-td** [nats-io/k8s#558] [Q&A] [answered]
Hi all,

I am looking to update my nats-operator installation so that I can support k8s 1.22+.

My current chart is the following:

```
repoURL: 'https://nats-io.github.io/k8s/helm/charts/'
targetRevision: 0.7.4
chart: nats-operator
```

I can't seem to find a `targetRevision` bigger than 0.7.4 anywhere. Can someone point me a more recent version?

Thanks!

**Comment by andretadeu** [comment, marked as answer]
Hi,

I believe you should check here: [NATS Operator repository](https://github.com/nats-io/nats-operator). After that, check if your have all the charts updated locally:

```
helm repo update
helm dependency update <Chart location>
```

**Reply by ekawas-td** [reply]
I think the bit that threw me off was that we get directed to https://github.com/nats-io/k8s/ when we try to get the chart from https://github.com/nats-io/nats-operator/tree/master/helm/nats-operator

The tooling I was using depended on the targetRevision and that doesn't appear to be up to date.
